### PR TITLE
Make create/delete stateful, fix ls

### PR DESCRIPTION
**Pull request by prog893**
- save/delete env info on create/delete commands (DynamoDB)
- small refactor on parsers
- fix setup.py requires
- fix ls output when there are no envs available
- update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,11 +103,11 @@ $ eden config -h
 usage: eden config [-h] {setup,check,push,remote-remove} ...
```
Comment (toVersus) [inline README.md]:
```suggestion
 usage: eden config [-h] {setup,check,push,remote-rm} ...
```

File: aws_eden_cli/dynamodb.py [hunk 2]
```diff
@@ -164,3 +165,22 @@ def fetch_all_environments(table):
             environments[profile_name].append(item)
 
     return environments
+
+
+def put_environment(table, profile_name, name, cname):
+    return table.put_item(
```
Comment (toVersus) [inline aws_eden_cli/dynamodb.py]: No exception handling here?

Comment (prog893) [reply aws_eden_cli/dynamodb.py]: fixed: e842917

**Review by toVersus — APPROVED** [review]
LGTM

**Pull request merged**